great_expectations/data_context/data_context.py

```diff
@@ -670,6 +672,78 @@ def update_return_obj(self, data_asset, return_obj):
         else:
             return return_obj
 
+    def profile_datasource(self, datasource_name, profiler_name="PseudoPandasProfiling", max_data_assets=10):
+        # logger.info("Profiling %s with %s" % (datasource_name, profiler_name))
+        print("Profiling %s with %s" % (datasource_name, profiler_name))
+        datasource = self.get_datasource(datasource_name)
```

Comment:
see `self.get_available_data_assets(datasource_name)`

Reply:
:+1:

Reply:
Noted for the merge refactor.